test/test.ml

```diff
@@ -1,23 +1,20 @@
-type ava
-type exe
+type t
 
-external t:ava = "ava" [@@bs.module]
+external ava:< 
+test : string -> (t -> unit) -> unit[@bs.meth]
```

Comment:
You don't really need this. If you use row polymorphism instead (`< .. >`) it will be inferred, which is usually good enough for internal bindings.